mibidata/tests/test_tiff.py

```diff
@@ -10,7 +10,7 @@
 import warnings
 
 import numpy as np
-from skimage import io, img_as_ubyte, transform
+from skimage import io as skio, img_as_ubyte, transform
```

Comment:
Why was it necessary to change this?

Reply:
I was trying to use the python `io` module, so I changed this one to `skio`. I didn't change it back because I thought it's clearer that it is not the standard python IO module. Let me know if you want it changed.